ocs_ci/ocs/ui/views.py

```
@@ -1885,6 +1885,16 @@
         "//*[@data-test-id='perspective-switcher-toggle']//h2[normalize-space()='Core platform']",
         By.XPATH,
     ),
+    "click-fleet-management": (
+        "//button[@data-test-id='perspective-switcher-toggle']"
+        "//*[normalize-space()='Core platform']",
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)